### ISSUE TITLE
Stop Loss

### ISSUE THREAD

**Issue by Orbitalz-eth** [bencagri/mokka#7]
Any chance you could add stop loss feature.

stop_loss = 1.0

so when we do a buy-limit order and the current price goes below the price we bought at; the bot will automatically sell.

**Comment by bencagri** [comment]
this is config which would be used in a strategy. it can be implemented.
